addons/project/models/res_company.py

```diff
@@ -15,11 +15,13 @@ class ResCompany(models.Model):
         help="Default Plan for a new analytic account for projects")
 
     def _compute_analytic_plan_id(self):
+        analytic_plan_project = self.env.ref("analytic.analytic_plan_projects")
+        AccountAnalyticPlan = self.env['account.analytic.plan']
         for company in self:
             default_plan = self.env['ir.config_parameter'].with_company(company).sudo().get_param("default_analytic_plan_id_%s" % company.id)
             company.analytic_plan_id = int(default_plan) if default_plan else False
             if not company.analytic_plan_id:
-                company.analytic_plan_id = self.env['account.analytic.plan'].with_company(company)._get_default()
+                company.analytic_plan_id = analytic_plan_project.id if analytic_plan_project.sudo().company_id == company else AccountAnalyticPlan.with_company(company)._get_default()
```

Comment:
@dhrs-odoo did you understand my previous comment? 🤔 

Because normally you no longer need to update that compute method. You have to do something else